src/mjolnir/valhalla_add_predicted_traffic.cc

```diff
@@ -61,7 +61,7 @@ struct TrafficSpeeds {
 };
 
 // Convert big endian bytes to little endian
-int16_t to_little_endian(const int16_t val) {
+int16_t to_little_endian(const uint16_t val) {
   return (val << 8) | ((val >> 8) & 0x00ff);
```

Comment:
yeah so bitshifting signed numbers is implementation dependent so im not sure what was happening here. i'm not sure if we have any negative numbers in the input but if we do this change will definitely have an impact on what we do with the data. i'll follow up with you offline with some sample data to see if we need to worry about it